src/mrb_capability.c

```diff
@@ -350,6 +460,12 @@ void mrb_mruby_capability_gem_init(mrb_state *mrb)
     mrb_define_method(mrb, capability, "getuid",        mrb_cap_getuid,      MRB_ARGS_NONE());
     mrb_define_method(mrb, capability, "getgid",        mrb_cap_getgid,      MRB_ARGS_NONE());
 
+    file = mrb_define_class_under(mrb, capability, "File", mrb->object_class);
```

Comment:
need to add `MRB_SET_INSTANCE_TT(file, MRB_TT_DATA)` for enable garbage collection of file ctx.